src/Support/SupportData.php

```diff
@@ -620,11 +652,98 @@ static function ( $url ) {
 					'urls'          => [],
 				];
 			}
+
+			// If no specific URL requested.
+			$query_args = [
+				'post_type'      => AMP_Validated_URL_Post_Type::POST_TYPE_SLUG,
+				'posts_per_page' => 100,
```

Comment:
@dhaval-parekh Why there's a posts limit set to 100?

Reply:
It's because there can be a lot of `amp_validated_post` posts and we shouldn't be fetching all those but also need to fetch a fair amount of data so the user can send it.